docker-compose.yml

```
@@ -7,6 +7,7 @@ services:
     volumes:
       - ./:/db
       - static_volume:/static
+      - image_volume:/images
```

Something like this should ensure that the images stay on the host disk if we delete the volumes:
```suggestion
      - ./images:/images
```
Make sure to test it though :)